src/components/user/TargetPoint.js

```diff
@@ -6,12 +6,12 @@ import twitterIcon from './../../assets/socialNetworks/twitter@3x.png';
 import playVideoIcon from './../../assets/phone/play@3x.png';
 
 const TargetPoint = () => (
-  <div>
+  <div className="targetPoint">
```

Comment:
Class names should be lowercase and hyphenated.

Reply:
I forgot this.